e2e/testdata/configsync-kcc/kcc/namespace.yaml

```diff
@@ -0,0 +1,21 @@
+# Copyright 2022 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: foo
+  annotations:
+    cnrm.cloud.google.com/project-id: "jingfang-fishfood"
```

Comment:
Should we update these names?

Reply:
hmm. I need to check if there's any prerequisite in that project.

I think that can be done in a separate PR.

Reply:
Ended up making the change in the same PR.
DONE